history/characters/59000_dalaranian.txt

```diff
@@ -336,7 +337,7 @@
 59160={
 	name=Daio
 	dynasty=36051
-	culture=dalaranian religion=arcane_religion
+	culture=dalaranian religion=dalaranian_religion
```

Comment:
Medivh (he's in a different file) should probably have this faith as well.

Reply:
Which Medivh? All of them? (Why are there so many???) Changed all of them tell me if I did it wrong.

Reply:
> Which Medivh? All of them? (Why are there so many???) Changed all of them tell me if I did it wrong.

That's real Medivh.